Tutorials/Strength_of_Simulation/Stance/Reference/leg69_Setup_CMC.xml

```diff
@@ -48,7 +48,7 @@
 		<desired_points_file>  </desired_points_file>
 		<!--Motion (.mot) or storage (.sto) file containing the desired kinematic
 		    trajectories.-->
-		<desired_kinematics_file> RRA/leg6dof9musc_Kinematics_q.sto </desired_kinematics_file>
+		<desired_kinematics_file> RRA/leg6dof9musc_RRA_adjusted_Kinematics_q.sto </desired_kinematics_file>
```

Comment:
@carmichaelong Is the new file already in repo/distribution or is it generated by the tutorial? Just making sure the tutorial files are self contained.

Reply:
Thanks for the careful review. Yes, I double checked and saw that I included this file (and the rest of the RRA output) here: https://github.com/opensim-org/opensim-models/blob/strength_of_simulation_kin_ext_load/Tutorials/Strength_of_Simulation/Stance/Reference/RRA/leg6dof9musc_RRA_adjusted_Kinematics_q.sto

This actually also fixes a problem where the previous file name didn't exist because the RRA files had a different prefix.